adsbibdesk.py

```diff
@@ -659,7 +650,7 @@ def has_annotationss(f):
     return sp.Popen(
         "strings %s | grep  -E 'Contents[ ]{0,1}\('" % f,
         shell=True, stdout=sp.PIPE,
-        stderr=open('/dev/null', 'w')).stdout.read() != ''
+        stderr=open('/dev/null', 'w')).stdout.read() != b''     # b''!=u'' in Python 3
```

Comment:
This is a pretty weird line to begin with.... why are we reading from stdout on `/dev/null` opened in write mode?

Reply:
Oh.... no, the `.stdout.read()` is on `sp.Popen`.  This is probably fine, but it would probably be better to put this into a context manager.